core/helpers/index.ts

```diff
@@ -43,11 +44,12 @@ export function getThumbnailPath (relativeUrl, width, height) {
     let baseUrl = processURLAddress(config.images.proxyUrl ? config.images.proxyUrl : config.images.baseUrl)
     if (baseUrl.indexOf('{{') >= 0) {
       baseUrl = baseUrl.replace('{{url}}', relativeUrl)
+      baseUrl = baseUrl.replace('{{type}}', type)
       baseUrl = baseUrl.replace('{{width}}', width)
       baseUrl = baseUrl.replace('{{height}}', height)
       resultUrl = baseUrl
     } else {
-      resultUrl = `${baseUrl}${parseInt(width)}/${parseInt(height)}/resize${relativeUrl}`
+      resultUrl = `${baseUrl}${type}/${parseInt(width)}/${parseInt(height)}/resize${relativeUrl}`
```

Comment:
This change assumes that you've got changed the `vs-api` config:  `imgUrl` (by removing `product` from it). it's not backward compatible change. We must probably have a `type=null` by default to support the old paths - the guys who have installed VS before this changes and have the path with `product` inside the `local.json`

Reply:
But here we could just check if the URL in the API has /product at the end when we load the config and remove it so we have the right URL.

Would that fix it?

Reply:
yes, probably it's a fix but it's very well specific one :) I mean - if somebody has been using differnt url strategy won't probably have `product` in there - so maybe chaning the deafult value is more general solution? No strong prefernce however.

Reply:
To fix it easily i would say i add a new config value to the config.images with which we can use the new logic. Which would fix the problem. 

So it would be the old way when the config is false and the new way when it is true. So the user needs to check if it is going to break stuff for him or not.

This would be a good way here and in the API.

Reply:
Yes, that makes sense for me